backend/src/main/scala/cromwell/backend/BackendWorkflowInitializationActor.scala

```diff
@@ -30,7 +30,7 @@ object BackendWorkflowInitializationActor {
   */
 trait BackendWorkflowInitializationActor extends BackendLifecycleActor with ActorLogging {
```

Comment:
Hmm, I think it's OK - it *can* be aborted but its *purpose* is to initialise the backend for the workflow